.snyk

```diff
@@ -35,4 +35,39 @@ ignore:
         reason: No fix available yet
         expires: 2024-10-11T00:00:00.000Z
         created: 2024-07-11T23:07:11.176Z
+  SNYK-GOLANG-K8SIOCLIENTGOTRANSPORT-7538822:
```

Comment:
Are these still needed with the 1.22 bump? Seems like the reason should be updated if so.

Reply:
Ah I was wrong, it is actually the dependencies fault when upgrading. I updated the error message